components/brave_new_tab_ui/containers/newTab/index.tsx

```diff
@@ -577,6 +593,30 @@ class NewTabPage extends React.Component<Props, State> {
     )
   }
 
+  renderBraveVPNWidget (showContent: boolean, position: number) {
+    const { newTabData } = this.props
+    const { showBraveVPN, textDirection, braveVPNSupported } = newTabData
+
+    if (!showBraveVPN || !braveVPNSupported) {
```

Comment:
optional nit: access directly if possible (might not be 'cause this is a class component).

If not, you could extract this function to a separate component, which takes props

Reply:
Done.